app/src/main/res/layout-land/question_player_fragment.xml

```diff
@@ -94,8 +94,33 @@
         android:progressDrawable="@drawable/progress_bar"
         app:layout_constraintBottom_toTopOf="@id/question_progress_text"
         app:layout_constraintEnd_toEndOf="parent"
-        app:layout_constraintStart_toStartOf="parent" />
-
+        app:layout_constraintStart_toEndOf="@id/hints_and_solution_fragment_container" />
+      <FrameLayout
+        android:id="@+id/hints_and_solution_fragment_container"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_gravity="bottom|start"
+        android:background="@drawable/hints_background"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintBottom_toBottomOf="parent"
+        android:visibility="@{viewModel.isHintBulbVisible() ? View.VISIBLE : View.GONE}">
+
+        <ImageView
+          android:id="@+id/dot_hint"
+          android:layout_width="8dp"
+          android:layout_height="6dp"
+          android:layout_gravity="top|end"
+          android:layout_margin="8dp"
+          android:src="@drawable/ic_dot_yellow_24dp"
+          android:visibility="@{viewModel.isHintOpenedAndUnRevealed() ? View.VISIBLE : View.GONE}" />
+
+        <ImageView
+          android:id="@+id/hint_bulb"
+          android:layout_width="wrap_content"
+          android:layout_height="wrap_content"
+          android:layout_margin="12dp"
+          android:src="@drawable/ic_hint_bulb_white_24dp" />
+      </FrameLayout>
```

Comment:
Any specific reason for shifting this entire framelayout above?

Reply:
Because the frame layout is outside the constraint layout’s domain,i am not able to add constraints to progress bar for views outside of the constraint layout.So i just added this inside the constraint layout

Reply:
Please see [this](https://github.com/oppia/oppia-android/pull/2540/files#r562765531)

Reply:
followed up in below comments.